examples/todos/src/actions/index.js

```diff
@@ -1,16 +1,33 @@
 let nextTodoId = 0
```

Comment:
Please add // @flow first for all file you wanna adding flow type. So, it can help you check error/missing flow type when running command `npm run flow`